src/models.py

```diff
@@ -0,0 +1,14 @@
+"""forum123's database models module."""
+
+from sqlalchemy import Column, Integer, String
+
+from src.database import Base
+
+
+class User(Base):  # pylint: disable=too-few-public-methods
+    """A model class for User database table."""
+
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True)  # noqa: A003
+    username = Column(String)
```

Comment:
We totally forgot to add database constraints to our fields. Here username should be`unique=True`, because we will use `username` field as a unique identificator for user, and `nullable=False` because we cannot have a user without `username`.
Yep, to accomplish that you need to do some routine again:
- run database
- revert migrations
- remove migration file
- add constraints to `models.py`
- generate new migration
- apply migration to database
- create two users
- check that `/index` works